src/qml/controls/BlockClock.qml

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2023 The Bitcoin Core developers
```

Comment:
Why should this be a control, and the block clock dial a component?

Reply:
This item can take focus and accept input.  The dial just renders the blocks.

Reply:
Actually not really convinced of this. Can see   it living in components and making sense too

Reply:
Really liked how you structured the commits in that branch. Going to use that.